spec/vcr_cassettes/manageiq/providers/openshift/container_manager/test_objects_record.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+set -e  # abort on error
+
```

Comment:
Since this isn't using functions can we have section headers for readability? e.g

```
#
# Ensure API Server
#
```

Reply:
Added an `echo`, I'm using these as both headings in code and headings in script output.